### PR TITLE
Add cargo nuget

### LINKED ISSUE #173: Move cargo nuget into this repo
[`cargo nuget`](https://github.com/rylev/cargo-nuget) can be used to install NuGet packages that work in the context of WinRT-rs. We should move that crate into this repository.

### PR DESCRIPTION
**Pull request by rylev**
Fixes #173

### CODE REVIEW

File: crates/cargo-nuget/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+use std::path::{Path, PathBuf};
+
+fn main() {
+    let Opt::Nuget { subcommand } = Opt::from_args();
+    match subcommand {
+        Subcommand::Install(i) => i.perform().unwrap(),
+    }
+}
+
+/// A utility for interacting with nuget packages
```
Comment (kennykerr) [inline crates/cargo-nuget/src/main.rs:17]: This comment seems out of place.

File: crates/cargo-nuget/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+            for dll in dep.dlls() {
+                dll.write(&dep_directory).unwrap();
+            }
+        }
+
+        Ok(())
+    }
+}
+
+fn workspace_root() -> PathBuf {
```
Comment (kennykerr) [inline crates/cargo-nuget/src/main.rs:74]: Maybe just remove this function for now?

File: crates/cargo-nuget/src/main.rs (hunk 1 above)
Comment (rylev) [reply crates/cargo-nuget/src/main.rs:17]: I believe `StructOpt` uses this to display helpful info the user.

File: crates/cargo-nuget/src/main.rs (hunk 2 above)
Comment (rylev) [reply crates/cargo-nuget/src/main.rs:74]: I'd rather just implement it :-)

**Pull request merged**